taipy/gui/extension/library.py

```diff
@@ -159,12 +159,17 @@ def _call_builder(
         properties: t.Optional[t.Dict[str, t.Any]],
         lib: "ElementLibrary",
         is_html: t.Optional[bool] = False,
-        counter: int = 0
+        counter: int = 0,
     ) -> t.Union[t.Any, t.Tuple[str, str]]:
         attributes = properties if isinstance(properties, dict) else {}
         if self.inner_properties:
             uniques: t.Dict[str, int] = {}
-            self.attributes.update(self.inner_properties)
+            self.attributes.update(
```

Comment:
is it to copy the object over and not have the same object

Reply:
inner properties use the default value of Element Property to hold the expression ... we don't want those strings has `real` default values